packages/genesys-spark-components/src/components/beta/gux-form/gux-form.light.scss

```diff
@@ -0,0 +1,97 @@
+gux-form-beta {
+  form {
+    max-width: var(--gse-semantic-forms-maxWidth);
+
+    header {
+      display: flex;
+      flex-direction: column;
+      gap: var(--gse-semantic-forms-formHeader-gap);
+      padding-bottom: var(--gse-semantic-forms-formHeader-paddingBottom);
+      text-align: left;
+
+      gux-form-heading {
+        margin: 0;
+        font-family: var(
+          --gse-semantic-forms-formHeader-text-fontFamily
+        ) !important;
+        font-size: var(
+          --gse-semantic-forms-formHeader-text-fontSize
+        ) !important;
+        font-weight: var(
+          --gse-semantic-forms-formHeader-text-fontWeight
+        ) !important;
+        line-height: var(
+          --gse-semantic-forms-formHeader-text-lineHeight
+        ) !important;
+        color: var(--gse-semantic-forms-formHeader-foregroundColor) !important;
```

Comment:
Why do we need to use !important for all of these?

Reply:
I think using `!important` is the only way to overide the default stylings of for example a `h1`. 
![Screenshot 2024-06-12 at 09 32 36](https://github.com/MyPureCloud/genesys-spark/assets/101644078/03e66273-c128-4030-b8ba-2813145804f1)
If I am not to use `!important` the default `h1` will take precedence as shown in the image above.

Reply:
Ah ok, cool. I had a look there and it looks to me like the style from the gux-form-heading component for slotted(*) takes precedence over this. Do both sets of styles have different purposes?

Reply:
The `::slotted(*)` styles are just specific to this design for the forms. The other `h1` styles are for if you were to use a `h1` in a component for example like the card component or something so they are aligned with the spark default styles.

Reply:
Sorry, what I'm trying to ask is if we have this coming from the component gux-form-header as in the image do we also need to have them in gux-form.light? 
![Screenshot 2024-06-12 at 14 02 41](https://github.com/MyPureCloud/genesys-spark/assets/25255508/446e39e6-494e-41e9-8dac-6bebe3596c75)

Reply:
Oh yeah, so the reasoning why we have this styling in two places is because the `:slotted(*)` selector in `gux-form-heading` cannot target text nodes so for example if you do something like,

`<gux-form-heading>Hello</gux-form-heading>` 

the `::slotted(*)` wont be able to target it. I think its a know issue but the only way around it is to also style it in the `gux-form.light.scss` file so we can cover both if the user slots a text node or slots and element.

Reply:
ah ok, that makes sense.